internal/btf/types.go

```diff
@@ -19,6 +20,16 @@ func (tid TypeID) ID() TypeID {
 
 // Type represents a type described by BTF.
 type Type interface {
+	// Type can be formatted using the %s and %v verbs. %s outputs only the
+	// identity of the type, without any detail. %v outputs additional detail.
+	//
+	// Use the '+' flag to include the address of the type.
+	//
+	// Use the width flag to specify how many levels of detail to output.
```

Comment:
This is genius.

We should probably give a quick example (e.g. `%5s`) as a small improvement. The number before the verb is technically not a flag, I think those are `+`, `-`, `#`, `' '` and 0.

Reply:
Thanks! I reworded the help text.